packages/code-du-travail-frontend/src/conventions/ConventionExplainer.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
+import { Section, Wrapper } from "@cdt/ui";
+import styled from "styled-components";
+
+const storedExpandedKey = "isConventionExplainerExpanded";
+
+const ConventionExplainer = () => {
+  const [expanded, setExpanded] = useState(true);
+
+  useEffect(() => {
+    const expanded = sessionStorage.getItem(storedExpandedKey) == "true";
+    setExpanded(expanded);
+  });
+
+  return (
+    <Section>
+      <Wrapper variant="dark">
+        <ExpandWrapper>
```

Comment:
Why did you not  use an `<Accordion />` here ?

Reply:
I didn't check that we had such a component 🤦‍♂ I'll change to use it